ui/app/AppLayouts/Chat/views/communities/CommunityNewPermissionView.qml

```diff
@@ -87,6 +87,48 @@ StatusScrollView {
             }
             property bool isPrivateDirty: false
 
+            function getIndexOfKey(key) {
```

Comment:
Does it make sense to have these functions in `CommunityPermissionHelpers` instead?

Reply:
I think it's quite tightly coupled with handling of dirty values so I decided to keep it here now. We can reconsider it when refactoring dirty values to be handled in a more systematic way.